src/pages/help-centre/_help-articles.js

```diff
@@ -109,6 +114,111 @@ const AuthenticatingYourAccount = () => (
     </Text>
 )
 
+const HowDoIChangeOrResetMyDerivPassword = () => (
+    <>
+        <Text>
+            {localize(
+                'To change your Deriv password if you’re already logged in:',
+            )}
+        </Text>
+        <Ol>
+            <Li>
+                <Localize
+                    translate_text="Go to <0>Settings > Security and safety > Deriv password.</0>"
+                    components={[<BoldText key={0} />]}
```

Comment:
you can use `<strong>` as well

Reply:
Sure. 
You are right. No need to write an extra component for it. Thank you.